docs/sources/reference/run.md

```diff
@@ -177,6 +177,19 @@ within the container.
 This command would allow you to use `strace` inside the container on pid 1234 on
 the host.
 
+## UTS settings (--uts)
+    --uts=""  : Set the UTS Namespace mode for the container,
```

Comment:
Could you add a blank line between the header and the code block here? For consistency, and I seem to remember the docs sometimes rendered incorrectly if it was missing (but not sure about that)